test/verify/check-networking

```diff
@@ -120,7 +120,7 @@ class TestNetworking(MachineCase):
 
         # Configure a manual IP address
         #
-        b.click("tr:contains('IPv4') button")
+        b.click("tr:contains('IPv4') a", True)
```

Comment:
The `a` element had illegal content, actually: a couple of `div`s.  I have fixed that.